### ISSUE TITLE
Rearranging the order of menus 

### ISSUE THREAD

**Issue by jackdougall1** [jackdougall1/servertrial#1]
I think it would be best if we rearrange the draw tool as well as the wind farm creation section as well. 

For the draw I think it's best to have the naming at the top, styling etc next and then the tools at the bottom as that's the order that they should be used in. 

for the wind farm layout draw i think the calculate yield and clear WT button should be at the bottom and select/draw function should be the last thing in the list as it being at the start makes the user want to select or draw straight away. 
basically the order i think it should be is: 
Draw mode
weather data
grid spacing 
draw polygon 

then underneath have a heading saying 'install turbines' 
then have 
Place 
Column spacing by diameter
row spacing by diameter 
no. of wt in column
no of wt in row
clear WT 
calculate yield 

quick question as well, what is the purpose of the second set of spacing options. is that not done with the grid spacing already?
